packages/reports/addon/components/filter-values/dimension-select.js

```diff
@@ -40,27 +40,27 @@ export default class DimensionSelectComponent extends Component {
   /**
    * @property {String} dimensionName - name of dimension to be filtered
    */
-  @readOnly('filter.subject.field') dimensionName;
+  @readOnly('filter.field') dimensionName;
 
   /**
    * @property {String} primaryKey - primary key for this dimension
    */
-  @readOnly('filter.subject.columnMetadata.primaryKeyFieldName') primaryKey;
+  @readOnly('filter.columnMetadata.primaryKeyFieldName') primaryKey;
 
   /**
    * @property {BardDimensionArray} dimensionOptions - list of all dimension values
    */
-  @computed('dimensionName', 'filter.subject.columnMetadata.cardinality', 'searchTerm')
+  @computed('dimensionName', 'filter.columnMetadata.cardinality', 'searchTerm')
   get dimensionOptions() {
     if (this.searchTerm !== undefined) {
       return undefined; // we are searching, so only show search results
     }
 
     const dimensionName = get(this, 'dimensionName'),
       dimensionService = get(this, '_dimensionService'),
-      source = get(this, 'filter.subject.source');
+      source = get(this, 'filter.source');
```

Comment:
Is using `ember.get` necessary?

```js
const { dimensionName, dimensionService: _dimensionService, filter: { source } } = this;
```

Reply:
Did you mean to have `_dimensionService: dimensionService` ? The way you had it comes up as dimensionService is not defined

Reply:
oops yeah that's what I meant